Src/Support/Google.Apis.Auth/SignedTokenVerificationOptions.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+Copyright 2020 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Util;
+using System;
+using System.Collections.Generic;
+using static Google.Apis.Auth.SignedTokenVerification;
+
+namespace Google.Apis.Auth
+{
+    /// <summary>
+    /// Options to use when verifying signed JWTs.
+    /// </summary>
+    public sealed class SignedTokenVerificationOptions
+    {
+        /// <summary>
+        /// Default verification options.
+        /// </summary>
+        public static SignedTokenVerificationOptions Default => new SignedTokenVerificationOptions();
+
+        internal SignedTokenVerificationOptions() { }
+
+        internal SignedTokenVerificationOptions(SignedTokenVerificationOptions other)
```

Comment:
This constructor could be public too.

(I know there are definitely benefits in using factory methods, but I don't think they're important here - whereas the use of object initializers would be nice.)